llvm/lib/ProfileData/InstrProfWriter.cpp

```diff
@@ -603,23 +592,17 @@ static Error writeMemProfV2(
 // uint64_t Schema entry N - 1
 // OnDiskChainedHashTable MemProfRecordData
 // OnDiskChainedHashTable MemProfFrameData
```

Comment:
As a side note, does this description need to be updated to add the MemProfCallStackData?

Reply:
Yes, thank you for pointing this out!  The comment needs to be updated in a version-specific manner.  Let me create a separate PR.